Realm/RLMRealm.h

```diff
@@ -495,11 +495,21 @@ __deprecated_msg("Use `performMigrationForConfiguration:error:`") NS_REFINED_FOR
  */
 + (BOOL)performMigrationForConfiguration:(RLMRealmConfiguration *)configuration error:(NSError **)error;
 
-- (void)dispatchAsync:(dispatch_queue_t)queue withBlock:(void(^)(RLMRealm *))block;
-- (void)dispatchAsync:(dispatch_queue_t)queue handingOverObject:(RLMObject *)objectToHandOver
-            withBlock:(void(^)(RLMRealm *, RLMObject *))block;
-- (void)dispatchAsync:(dispatch_queue_t)queue handingOverObjects:(NSArray<RLMObject *> *)objectsToHandOver
-            withBlock:(void(^)(RLMRealm *, NSArray<RLMObject *> *))block;
+- (void)dispatchAsyncWithBlock:(void(^)(RLMRealm *))block
```

Comment:
I updated the Objective-C methods to follow the naming conventions @jpsim suggested.